package_test.go

```diff
@@ -1321,3 +1321,63 @@ func (s *PackageSuite) TestIterMethodOrder(c *C) {
 	err = iter.Close()
 	c.Assert(err, IsNil)
 }
+
+type PersonAutoID struct {
+	ID         int    `db:"id,omitempty"`
+	Fullname   string `db:"name"`
+	PostalCode int    `db:"postal_code"`
+}
+
+var fredAutoId = PersonAutoID{Fullname: "Fred", PostalCode: 1000}
+var markAutoId = PersonAutoID{Fullname: "Mark", PostalCode: 1500}
+var maryAutoId = PersonAutoID{Fullname: "Mary", PostalCode: 3500}
+var daveAutoId = PersonAutoID{Fullname: "James", PostalCode: 4500}
+var allPeopleAutoId = []PersonAutoID{fredAutoId, markAutoId, maryAutoId, daveAutoId}
+
+func (s *PackageSuite) TestOmitEmpty(c *C) {
+	db, err := createTestDB()
+	c.Assert(err, IsNil)
+
+	createPerson, err := sqlair.Prepare(`
+		CREATE TABLE personAutoID (
+			id INTEGER PRIMARY KEY AUTOINCREMENT,
+			name text,
+			postal_code integer
+		);
+	`)
+	c.Assert(err, IsNil)
+	err = db.Query(nil, createPerson).Run()
+	c.Assert(err, IsNil)
+	defer dropTables(c, db, "personAutoId")
```

Comment:
Should that string be "personAutoID"? Seems like SQLite is case insensitive, but still worth changing.

Reply:
Nice catch.